go/vt/vttablet/tabletserver/vstreamer/uvstreamer.go

```diff
@@ -253,9 +253,15 @@ func (uvs *uvstreamer) filterEvents(evs []*binlogdatapb.VEvent) []*binlogdatapb.
 		}
 		if !shouldSend && tableName != "" {
 			shouldSend = true
-			_, ok := uvs.plans[tableName]
+			plan, ok := uvs.plans[tableName]
 			if ok {
-				shouldSend = false
+				// Ideally we should compare the PKs and never send the rows whose PK has already been copied.
+				// For now, we send all the changes by accepting the duplicate events.
```

Comment:
[Question] I'm not sure this compromise is acceptable in general.
I guess there is already a discussion about the balance between the accuracy and the performance.

For example: https://github.com/vitessio/vitess/blob/3f25ae8f9e26493c92f03b984231b7f31ed1529e/go/vt/vttablet/tabletmanager/vreplication/replicator_plan.go#L253-L269

Reply:
Sifting through like "pk <= lastpk" without any negative performance impact is ideal.